lib/pluginOverride.js

```diff
@@ -48,8 +49,14 @@ module.exports = function override (old, fn, opts) {
   instance[kSchemaController] = SchemaController.buildSchemaController(old[kSchemaController])
   instance.getSchema = instance[kSchemaController].getSchema.bind(instance[kSchemaController])
   instance.getSchemas = instance[kSchemaController].getSchemas.bind(instance[kSchemaController])
-  instance[pluginUtils.registeredPlugins] = Object.create(instance[pluginUtils.registeredPlugins])
-  instance[kPluginNameChain] = [pluginUtils.getPluginName(fn) || pluginUtils.getFuncPreview(fn)]
+
+  // Track the registered and loaded plugins since the root instance.
+  // It does not track the current encapsulated plugin.
+  instance[pluginUtils.kRegisteredPlugins] = Object.create(instance[pluginUtils.kRegisteredPlugins])
```

Comment:
Shouldnt kRegisteredPlugins be in symbols.js?

Reply:
I though the same 👍🏼

Reply:
Will you move it to there?